postapi/WeatherForecast.cs

```diff
@@ -0,0 +1,15 @@
+using System;
```

Comment:
This file also not needed, please remove.

Reply:
Updated the Changes